articles/templating-your-static-site/index.md

```diff
@@ -0,0 +1,188 @@
+
+Congratulations, you've just created your first Eleventy layout file.
+
+###  Rendering Your First Layout File
+
+We have our basic layout template but we need to tell Eleventy where to use it. This is where front-matter comes in.
+
+Front-Matter allows you to define variables you can use in your layout templates both built-in (from Eleventy) and those you've created yourself. By default, Eleventy uses YAML for front-matter though you can change it to JSON or even JavaScript if you prefer.
+
+The most basic front-matter variable, layout will instruct Eleventy which layout file to use to render a page.
+
+On one of the HTML pages of your site, remove the code you included in EJS partials (head, header and footer if you followed the tutorial.)
+
+Then, add the following layout front-matter at the beginning:
+
+```yaml
+---
+layout: _base-layout.ejs
+---
+```
+The three dashes define the start and the end of the front-matter.
+
+Run `npx @11ty/eleventy --serve` in the terminal to start a local web-server for your site that will reload on changes.
+
+Go to `http://localhost:8080/nameofhtmlfile` to view the page. It should display the head, header and footer as well as the content of the page.
+
+Congratulations, you've successfully converted your first static HTML page to use an EJS Eleventy layout file.
+
+## Adding More Front Matter
+
+Remember how we templated your website's header so that all the meta tags would be the same? Now that you've added your first piece of front-matter, we can add some more to account for unique values like different page titles.
+
+First, In your partial pages (such as `_head.ejs`), remove everything inside the content attribute of your meta tags and replace it with EJS variables like the example below:
+
+```html
+<meta property="og:title" content="<%- title %>">
+<meta property="og:description" content="<%- short_description %>">
+```
+Adding - after <% definites it as an EJS variable rather than a partial file which we did before. The values such as title and short-description are the names of the variables which we will use in the front-matter.
+
+Next, in one of your HTML pages, add the EJS variables you've just created and the content you want to add like so:
+```yaml
+---
+layout: _base-layout.ejs
+title: About Louise Findlay - Front-End Web Developer
+short_description: Louise Findlay has designed and developed over 20 static websites.
+---
+```
+Finally, check on your website and the front-matter content you've just added should be reflected in the code. If not, make sure you've run `npx @11ty/eleventy --serve` in the terminal to make your website automatically reload on changes.
+
+### Creating a Page Template (Layout Chaining)
+
+You've just created your basic layout containing the elements present on every page but what if you have a custom layout for only a select number of pages such as for products? This is where layout chaining comes in.
```

Comment:
"You've just created your basic layout containing the elements present on every page but what if you have a custom layout for only a select number of pages such as for products?" This sentence is awkward and missing some comas, consider revising entirely.

Reply:
Revised wording.